server/plugin/api.go

```diff
@@ -570,9 +574,21 @@ func (p *Plugin) handleDriveWatchNotifications(c *Context, w http.ResponseWriter
 			continue
 		}
 
-		modifiedTime, _ := time.Parse(time.RFC3339, change.File.ModifiedTime)
-		lastChangeTime, _ := time.Parse(time.RFC3339, change.Time)
-		viewedByMeTime, _ := time.Parse(time.RFC3339, change.File.ViewedByMeTime)
+		modifiedTime, err := time.Parse(time.RFC3339, change.File.ModifiedTime)
+		if err != nil {
+			p.API.LogError("Failed to parse modified time", "err", err, "userID", userID)
```

Comment:
nit: Add the failed datetime value to a log field so we can debug easily just checking the logs

Reply:
+1

Reply:
Done